src/Client.js

```diff
@@ -100,24 +102,22 @@ class Client extends EventEmitter {
             timeout: 0,
         });
 
-        const KEEP_PHONE_CONNECTED_IMG_SELECTOR = '[data-asset-intro-image-light="true"], [data-asset-intro-image-dark="true"]';
-
-        if (this.options.session) {
-            // Check if session restore was successfull 
+        const INTRO_IMG_SELECTOR = '[data-testid="intro-md-beta-logo-dark"], [data-testid="intro-md-beta-logo-light"]';
```

Comment:
it feels like we can pretty easily support non-md-enabled users by matching the old `[data-asset-intro-image-light="true"], [data-asset-intro-image-dark="true"]` in this selector as well. Is that right?

If simple to get it working for both users we should try to do that

Reply:
yes, it's simple

Reply:
but we would end up saving chrome data when we only need the session's JSON

Reply:
done anyways